android/src/main/java/com/magicleap/magicscript/scene/nodes/layouts/manager/GridLayoutManagerImpl.kt

```diff
@@ -182,4 +230,20 @@ class GridLayoutManagerImpl : GridLayoutManager {
         }
     }
 
+    private fun calculateMaxChildWidth(childIdx: Int): Float {
+        return if (parentWidth == WRAP_CONTENT_DIMENSION) {
+            Float.MAX_VALUE
+        } else {
+            maxChildWidthInColumnMap[getColumnIndex(childIdx)]!!
```

Comment:
are you sure about `!!`?

Reply:
Yes, I'm sure. This map is filled with correct values always before this method execution. We use also use map with children bounds and !!, but always we are sure about values :-)